src/chromatix/functional/propagation.py

```diff
@@ -144,70 +162,71 @@ def exact_propagate(
     return field
 
 
-def propagate(
-    field: Field,
-    z: float,
-    n: float,
-    *,
-    method: str = "transfer",
-    mode: str = "full",
-    N_pad: Optional[int] = None,
-    loop_axis: Optional[int] = None,
-) -> Field:
+def calculate_padding_transform(
+    height: int, spectrum: float, dx: float, z: float
+) -> int:
+    """
+    Automatically calculate the padding required for transform propagation.
+
+    Args:
+        height: Shape of the field
```

Comment:
This should say height of the field

And we should have a TODO that this only works on square fields for now

Reply:
done